### PR TITLE
Bug 1187891: New Tabzilla V0.3.0

### PR DESCRIPTION
**Pull request by stephaniehobson**
Updated HTML to new snippet.

Updated CSS to new CSS, added solid background for default pages and over-ride to transparent on zone pages.

Remove async call to load tabzilla assets.

Added tabzilla images to our code base.

Not sure if this is 100% of what we have to do to complete the bug. May also need to change how we host/call the font files.

### CODE REVIEW

File: media/stylus/components/structure/tabzilla.styl [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
+  height: 37px;
+  text-indent: 120%;
+  white-space: nowrap;
+  overflow: hidden;
+  background-image: url('/media/img/tabzilla-static.png');
+  background-repeat: no-repeat;
+  z-index: 2;
+}
+@media (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (min-resolution: 1.5dppx) {
+  #tabzilla a {
```
Comment (mdn-services) [inline media/stylus/components/structure/tabzilla.styl]: Media Queries: resolution feature not supported by: IE (10,11), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

Comment (stephaniehobson) [inline media/stylus/components/structure/tabzilla.styl]: Oh hai mdn-services!

Comment (darkwing) [inline media/stylus/components/structure/tabzilla.styl]: :D

Comment (groovecoder) [inline media/stylus/components/structure/tabzilla.styl]: :clap:

Also, I filed https://github.com/mozilla/tabzilla/issues/62 upstream.

Comment (stephaniehobson) [inline media/stylus/components/structure/tabzilla.styl]: I tested in Safari and it still rendered the high res one okay. Worst case scenario: IE gets the non high-res version. Progressive enhancement FTW!

**Comment by darkwing** [comment]
R+!  Nice!

File: media/stylus/components/structure/tabzilla.styl (hunk 1 above)
Comment (mdn-services) [inline media/stylus/components/structure/tabzilla.styl]: Media Queries: resolution feature not supported by: IE (10,11), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

**Pull request merged**